webcentral/src/pages/views.py

```diff
@@ -534,12 +534,12 @@ def criteriaCatalog(request):
     explanText += _(
         "<p>Anmerkung: Der Kriterienkatalog wird aktuell noch laufend ergänzt und bearbeitet. Bei allen Empfehlungen handelt es sich um typisierte Beispiele, die im Lichte des konkreten Anwendungsfalls betrachtet und umgesetzt werden müssen.</p>"
     )
-    explanTextRendered = Template(explanText).render(Context({}))
+    # explanTextRendered = Template(explanText).render(Context({}))
```

Comment:
Can this line be deleted?

Reply:
i would keep it, maybe we need it in the future